### ISSUE TITLE
Massive chunk rendering problems (2-3 fps max)

### ISSUE THREAD

**Issue by xGarionx** [scalda/FoolCraft#594]
## SUGGESTION/HELP
- [ ] I would like to propose an additional mod, config change, custom recipe, etc.
- [x] I am requesting help with installing or running the pack

While trying to play a world the fps immediatly drops to 2-4 fps at max. Chunks get loaded correctly but not render unless the [ESC] Button is pressed. I really would love to play the pack but its sadly not working this way.

## Reproduce Steps
- Simply start a game and the fps instantly goes down to 2-3fps
- The chunks get loaded/rendered instantly on hitting [Esc] and with that the fps goes up to around 170 
- trying to play the game pushes the fps back to 2-3 in a mere second

## Additional Information

Stuff i tried sofar that didnt help:
-> Deactivate Biomes o plenty  
-> reducing loading chunks and graphics to min or maximum makes no difference
-> made sure minecraft uses the GPU as it should
-> installing ,reainstalling and rereinstalling the modpack
-> tried 1,4 version and 1.3 version with no difference
-> tried to increase and lower the run (from 2gb to 12 gb in 2gb steps no change)

System : 
- Win 10 64x
- 32 gb Ram (
-> Amd 8 core (4ghz)
-> Radeon RX 480 8 Gb version

------------------------
I really hope someone has a suggestion to help me ,would love to play the hell out of this pack.

**Comment by xGarionx** [comment]
Little Update:
Tried a flatworld with no structures aswell , interessting enough the world generation ignores it completly.
However i loaded a flatworld from hermitcraft with standart superflat and no structrues.
Issues stays the same

**Comment by csokasg** [comment]
i only experienced this if i gave mc less than 4 gb ram, and it ran out of memory constantly

**Comment by xGarionx** [comment]
as mentioned earlier changing the ram had no effect at all . I started with 6 GB of Ram , knowing minecraft can act strangely i tried it with the minium amount of ram that is needed (so 4 gb) ,than with no changes i increased the ram everytime by 2 Gb up to 12 GB (couldnt allocate more) no change.

**Comment by xGarionx** [comment]
As a little update. Modpacks like Ftb beyond work fine with constant 60 fps on 16 chunk loading with full graphics, it is a little bit bigger than foolcraft (not as good balancing and fun wise... not even close), so i hereby rule out the pc ,or graphics settings.

**Comment by xGarionx** [comment]
Annnd another update, I reinstalled again and again it seems 1.3.3 is working just fine, 30~65 fps . For reference 1.4.8 still isnt working.

**Comment by irobinson107** [comment]
I am having the exact same issue in 1.4.8 with Minecraft in launcher set to 12GB and have 24GB in pc.  Hit ESC and FPS goes way up to 100+.  Can get FoolCraft v1.3.3 or 1.2.1 to run normally.

**Comment by xGarionx** [comment]
I was going crazy thinking that there is something competly broken good to know someone else have the same issue, so now its safe to asume, that there is some issue between 1.3.3 and 1.4.8, in chunk rendering regards.

**Comment by irobinson107** [comment]
I figured out the issue, since it was happening on over 90% of any of the Modded Minecraft through the Twitch launcher both in Single Player and Multiplayer.  It was my AMD Graphics Card in my Alienware Aurora.  All of the other players on my sever over 12 did not have the issue and all had Nvdia or Intel cards on laptops and could run 60 FPS in Fast no problem.  I just swapped my AMD 2GB card for a Nvdia Titan Ti 1050 4GB card and can run 60 FPS in Fancy.  Hope this helps.
